MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1129,6 +1206,13 @@
  browser's navigation bar. The button it labels, when pressed, Shows the WebXR permission dialog. -->
     <string name="webxr_blocked_tooltip">WebXR Blocked</string>
 
+    <!-- This string appears in the WebXR interstial next to an arrow pointing to the controller button that
+     has to be used to exit from WebXR. It's preferred to separate it in two lines if possible. -->
+    <string name="webxr_interstitial_exit_webxr">Exit\nWebXR&quot;</string>
```

Comment:
@MortimerGoro  @keianhzo 
This doesn't look right? Why do you have a stray double quote at the end of the string?

Also, can you add in the comment that "\n" is used to create a new line?